.circleci/config.yml

```diff
@@ -83,15 +171,17 @@ workflows:
   version: 2
   test:
     jobs:
+      - go-fmt-and-vet
+      - unit-acceptance-framework:
+          requires:
+            - go-fmt-and-vet
       - unit-helm2
```

Comment:
Do you care to have the fmt/vet checks pass before running the helm2/3 unit tests? ie: also require `go-fmt-and-vet` for these 2 jobs

Reply:
We don't at the moment since they are using `bats`, so we don't need to require it (at least for now 😃 ).